spec/integration/container/plugins/zeitwerk/namespaces_spec.rb

```diff
@@ -1,9 +1,7 @@
 # frozen_string_literal: true
 
 RSpec.describe "Zeitwerk plugin / Namespaces" do
-  include ZeitwerkHelpers
-
-  after { teardown_zeitwerk }
+  after { ZeitwerkLoaderRegistry.clear }
```

Comment:
Do the loaders created in this example group get torn down? Here we're letting the plugin create the loader itself (which is just a plain old `::Zeitwerk::Loader.new`), rather than assigning a loader that we've created ourselves via `ZeitwerkLoaderRegistry.new_loader`.

(This question applies to any of the spec files that follow a similar approach to this one)

Reply:
Hmm, good point. One thing to remedy this could be to just stub `Zeitwerk::Loader.new` in tests with `ZeitwerkLoaderRegistry.new_loader`, but I'm not sure how we feel about this.

Reply:
It is done now. Sorry it took this long.